6_handbook/20_application_tuning.txt

```diff
@@ -1,32 +1,27 @@
-
 === Application tuning
 
-Some applications used by Rudder can be tuned to your needs, like Apache HTTPd.
+Some applications used by Rudder can be tuned to your needs, like the Apache web
+server, or Jetty.
 
-==== Apache HTTPd
+==== Apache web server
 
-The apache HTTPd is used by Rudder as a proxying server, to connect to the Jetty
-application server.
+The Apache web server is used by Rudder as a proxy, to connect to the Jetty
+application server, and to receive inventories using the WebDAV protocol.
 
-But it is also widely used as a regular HTTP serving application. You are heavily
-advised if interested to read the tons of documentation about it in your Linux
-distribution website, to learn about what it can do.
+There are tons of documentation about Apache performance tuning available on the
+Internet, but the defaults should be enough for most setups.
 
 ==== Jetty
 
-The Jetty 7 (Hightide) application server is the main application that runs the
-Rudder code. It is based on the Java programming language.
-
-About the latter, there is some configuration switches that you might want to
-tune to obtain better performance with Rudder, in /opt/rudder/etc/rudder-jetty.conf, whereas
-the default ones fit the basic recommendations for the minimal Rudder hardware requirements.
+The Jetty application server is the service that runs Rudder web application and inventory
+endpoint. It uses the Java runtime environment (JRE).
 
-* -Xms and Xmx: These parameters tune the total amount of RAM usable / dedicated to the
-java process. It is what you want to tune at first to give Rudder some more RAM.
```

Comment:
We explain how to do it in the +/etc/default/rudder-jetty+ file, mentioned 3 lines below